app/helpers/navigation_helper.rb

```diff
@@ -5,11 +5,37 @@ def active_link_for(segment, trainee = nil)
     url = request.path_info
 
     {
-      drafts: -> { trainee&.draft? && segment == "drafts" },
-      trainees: -> { url.include?("/#{segment}") && !trainee&.draft? },
-      bulk: -> { url.include?(segment) },
+      drafts: -> { trainee&.draft? && segment == :drafts },
+      trainees: -> { url.include?("/trainees") && !trainee&.draft? },
+      bulk: -> { url.include?("bulk") },
       reports: -> { url.include?("reports") },
-      funding: -> { ["/funding/monthly-payments", "/funding/trainee-summary"].include?(url) },
-    }[segment.to_sym].call
+      funding: -> { ["/funding/monthly-payments", "/funding/trainee-summary", "/system-admin/fundings"].include?(url) },
+    }[segment].call
+  end
+
+  def navigation_items(trainee, current_user)
+    [
+      { name: "Home", url: root_path },
+      ({ name: "Draft trainees", url: drafts_path, current: active_link_for(:drafts, trainee) } if can_view_drafts?),
+      { name: "Registered trainees", url: trainees_path, current: active_link_for(:trainees, trainee) },
+      ({ name: "Bulk updates", url: bulk_update_path, current: active_link_for(:bulk) } if can_bulk_update?),
+      ({ name: "Reports", url: reports_path, current: active_link_for(:reports) } if can_view_reports?),
+      funding_navigation_item,
```

Comment:
i think this should go in the sys admin nav rather than the main nav - it's essentially very similar functionality to 'Uploads'.

@andrewscrivener the prototype only has one nav which combines the two, but this is how the two look on the site:

![image](https://github.com/DFE-Digital/register-trainee-teachers/assets/168365/2870afc0-83f6-41f0-a441-093da10846e8)

Reply:
Yes I understand that prod and the prototype differ currently @carlosmartinez but surely we should be moving to where the UI needs to be from a users perspective rather than wrapping tabs in a single view.

<img width="1130" alt="Support UI" src="https://github.com/DFE-Digital/register-trainee-teachers/assets/1358700/350a0023-abb6-4c9f-a9e7-8a74c6d0dbe0">

Reply:
Shall we discuss with Sarah on Monday? It seems odd to me that sysadmin users would go to the sysadmin area for everything except this one thing (funding uploads), but anyway, it's easy enough to change this if we need to.

Reply:
Menu item has moved 
![image](https://github.com/DFE-Digital/register-trainee-teachers/assets/470137/f252c2a6-5a38-46bf-a122-baa65f1e33ba)